src/js/components/pages.js

```diff
@@ -384,10 +393,31 @@ export class NoticeIndexPage {
     }
   }
 
+  renderFeatured() {
+    const featuredNotices = this.renderChildPageLinks(true);
+
+    if (featuredNotices.length > 0) {
```

Comment:
Just a personal preference but, what if we return early if there are no notices i.e.

```javascript
if (!featuredNotices.length) {
  return [];
}
```

Reply:
sounds good!